Dev fp vs oop/README.md

```diff
@@ -0,0 +1,107 @@
+
+## Example of the same code used in functional programming 
+public class NumberManipulator {
+    private List<Integer> numbers;
+
+    public NumberManipulator(List<Integer> numbers) {
+        this.numbers = numbers;
+    }
+
+    public List<Integer> squareNumbers() {
+        List<Integer> squaredNumbers = new ArrayList<>();
+        for (int num : numbers) {
+            squaredNumbers.add(num * num);
+        }
+        return squaredNumbers;
+    }
+
+    public List<Integer> getEvenNumbers() {
+        List<Integer> evenNumbers = new ArrayList<>();
+        for (int num : numbers) {
+            if (num % 2 == 0) {
+                evenNumbers.add(num);
+            }
+        }
+        return evenNumbers;
+    }
+
+    public int sumNumbers() {
+        int sum = 0;
+        for (int num : numbers) {
+            sum += num;
+        }
+        return sum;
+    }
+
+    public static void main(String[] args) {
+        // Sample list of numbers
+        List<Integer> numbers = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
+        NumberManipulator manipulator = new NumberManipulator(numbers);
+
+        // Example 1: Square each number
+        List<Integer> squaredNumbers = manipulator.squareNumbers();
+        System.out.println("Squared numbers: " + squaredNumbers);
+
+        // Example 2: Select even numbers
+        List<Integer> evenNumbers = manipulator.getEvenNumbers();
+        System.out.println("Even numbers: " + evenNumbers);
+
+        // Example 3: Sum all numbers
+        int sum = manipulator.sumNumbers();
+        System.out.println("Sum of numbers: " + sum);
+    }
+}
+
+## Conclusion: 
+In conclusion, functional programming is about handling data in a straightforward and predictable way, while object-oriented programming is about organizing code into reusable parts, like building blocks. Functional programming is great for tasks involving lots of data, while object-oriented programming is better for building bigger, more complex systems. By using both approaches together, developers can make software that's flexible and works well for different kinds of projects.
+
+## References:
+* https://fluxtech.me/blog/object-oriented-programming-vs-functional-programming/#:~:text=Key%20differences%20between%20the%20two%20paradigms&text=FP%20focuses%20on%20functions%20and,%2Dorder%20functions%2C%20and%20recursion.
+
+* https://careerfoundry.com/en/blog/web-development/functional-programming-vs-oop/
```

Comment:
References in README files should not be bare links. Please rewrite in a more essay-oriented style, with author name and article title listed.